files/default/neutron-ha-tool.py

```diff
@@ -922,6 +925,41 @@ def pick(self):
         return random.choice(self.agents)
 
 
+class LeastBusyAgentPicker(object):
+    def __init__(self, qclient, agents):
+        self.now = datetime.datetime.now
```

Comment:
Is it good practice for a class to "steal" methods from another class like this?  I don't know, but I'm dubious it is, because it changes the context in which the method is executed.  If the method relies on internal state (which it probably doesn't in this case) then couldn't that be evaluated in the wrong context?  I would have thought that

    def now(self):
        return datetime.datetime.now()

would be safer and less voodoo, but I'd be happy to hear that I'm wrong.